code/__HELPERS/unsorted.dm

```diff
@@ -1602,3 +1602,14 @@ config_setting should be one of the following:
 /proc/get_final_z(atom/A)
 	var/turf/T = get_turf(A)
 	return T ? T.z : A.z
+
+/proc/invertDir(var/input_dir)
+	switch(input_dir)
+		if(NORTH)
+			return SOUTH
+		if(SOUTH)
+			return NORTH
+		if(EAST)
+			return WEST
+		if(WEST)
+			return EAST
```

Comment:
```suggestion
/proc/invertDir(var/input_dir)
	switch(input_dir)
		if(NORTH, SOUTH)
			CRASH("Can't turn vertical directions!")
		else return turn(input_dir, 180)
```

Reply:
that works, but I'll make it invert up/down too xd